src/Generator.cpp

```diff
@@ -989,10 +990,11 @@ int generate_filter_main_inner(int argc, char **argv, std::ostream &cerr) {
         Target gcd_target = targets[0];
         for (size_t i = 1; i < targets.size(); i++) {
             if (!gcd_target.get_runtime_compatible_target(targets[i], gcd_target)) {
-                user_error << "Failed to find compatible runtime target for "
-                           << gcd_target.to_string()
-                           << " and "
-                           << targets[i].to_string() << "\n";
+                cerr << "Failed to find compatible runtime target for "
```

Comment:
Nit: this is easily confused for `std::cerr`. Should probably rename the parameter to `os` or something else conventional, but not clashing with a standard name.

Reply:
Done